lib/tasks/journeys_report.rake

```diff
@@ -97,24 +49,36 @@ namespace :journeys do
           from: move.from_location.title,
           to: move.to_location&.title,
           events: move.move_events.default_order.map do |move_event|
-            {
-              timestamp: move_event.client_timestamp.strftime('%d/%m %H:%M:%S'),
-              event: move_event.event_name,
-              notes: move_event.notes,
-              from_location: move_event.from_location&.title,
-              to_location: move_event.to_location&.title,
-            }
-          end,
+                    {
+                      timestamp: move_event.client_timestamp.strftime('%d/%m %H:%M:%S'),
+                      event: move_event.event_name,
+                      notes: move_event.notes,
+                      from_location: move_event.from_location&.title,
+                      to_location: move_event.to_location&.title,
+                    }
+                  end,
           journeys: journeys,
         }
       end
       suppliers << {
         supplier: supplier.key,
         moves: moves,
       }
-    end
-    File.open('journeys_report.json', 'w') do |file|
-      file.write(JSON.pretty_generate({ data: suppliers }))
+
+      File.open(filename, 'w') do |file|
+        file.write(JSON.pretty_generate({ data: suppliers })) # maybe change to .to_json for production
+      end
+
+      puts "Exporting #{filename} to S3..."
+      if ENV['S3_REPORTING_BUCKET_NAME']
+        bucket_name = ENV.fetch('S3_REPORTING_BUCKET_NAME')
```

Comment:
no need to check the ENV twice, how about
```
bucket_name = ENV.fetch('S3_REPORTING_BUCKET_NAME')
if bucket_name.present?
   ...
```

Reply:
Thanks! Will address that.